addons/hr_timesheet/migrations/10.0.1.0/pre-migration.py

```diff
@@ -7,6 +7,9 @@
 COLUMN_RENAMES = {
     'account_analytic_account': [
         ('use_timesheets', None),
+    ],
+    'account_analytic_line': [
+        ('is_timesheet', None),
```

Comment:
Why are you renaming this column? Isn't it used in new version as well?

Reply:
Although your logic on renaming can apply, please leave it as it was, as there may be other scripts that use this field with the previous field name, and you are breaking them.